Bugzilla/User.pm

```diff
@@ -2407,13 +2409,24 @@ sub is_insider {
 sub is_global_watcher {
     my $self = shift;
 
-    if (!defined $self->{'is_global_watcher'}) {
-        my @watchers = split(/[,;]+/, Bugzilla->params->{'globalwatchers'});
-        $self->{'is_global_watcher'} = scalar(grep { $_ eq $self->login } @watchers) ? 1 : 0;
+    if (!exists $self->{'is_global_watcher'}) {
+        my @watchers = split(/\s*,\s*/, Bugzilla->params->{'globalwatchers'});
+        $self->{'is_global_watcher'} = (any { $_ eq $self->login } @watchers) ? 1 : 0;
     }
     return  $self->{'is_global_watcher'};
 }
 
+sub is_silent_user {
+    my $self = shift;
+
+    if (!exists $self->{'is_silent_user'}) {
+        my @helpers = split(/\s*,\s*/, Bugzilla->params->{'silent_users'});
+        $self->{'is_silent_user'} = (any { $self->login eq $_ } @helpers) ? 1 : 0;
```

Comment:
`helpers` is a weird name to use here, instead of something like `users`.

Reply:
woops. Originally it was "global_helpers" before I settled on silent_users